4-toolchain/Dockerfile.rhel7

```diff
@@ -52,12 +51,12 @@ WORKDIR ${HOME}
 
 # Use entrypoint so path is correctly adjusted already at the time the command
 # is searching, so something like docker run IMG gcc runs binary from SCL.
-COPY contrib/bin/container-entrypoint /usr/bin/container-entrypoint
+COPY root/opt/app-root/bin/container-entrypoint /usr/bin/container-entrypoint
 
 # Install the usage script with base image usage informations
-COPY contrib/bin/usage /usr/local/bin/usage
+COPY root/opt/app-root/bin/usage /usr/local/bin/usage
 
-COPY contrib/etc/scl_enable /opt/app-root/etc/scl_enable
+COPY root/opt/app-root/etc/scl_enable /opt/app-root/etc/scl_enable
```

Comment:
Here I was talking only about the highlighted `/opt/app-root-etc/scl_enable` COPY instruction, which can be safely removed.

With `usage` and `container-entrypoint`, we have to decide if we need them at `/usr/(local/)bin` and if so, whether to do it using a Docker instruction like it's already done, or maybe a symlink inside the `root` subdirectory of this repo.